src/main/java/org/jenkinsci/plugins/DependencyTrack/JobAction.java

```diff
@@ -58,9 +58,9 @@ public String getUrlName() {
      */
     @SuppressWarnings("unused") // Called by jelly view
     public boolean isTrendVisible() {
-        final List<? extends AbstractBuild<?, ?>> builds = project.getBuilds();
+        final List<? extends Run<?, ?>> builds = project.getBuilds();
```

Comment:
AbstractBuild is a subclass of Run, but some things want Run, so use superclass.